File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int res[] = new int[nums.length];
        int product = 1;
        for(int i=0;i<nums.length;i++){
            for(int j=0;j<nums.length;j++){
                if(i!=j)
                product = product * nums[j];
            }
            res[i] = product;
            product = 1;
        }
        return res;
    }
}  

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        if k > len(nums):
            return []
        
        i = 0
        j = k-1
        result = []
        while j<len(nums):
            max_value = max(nums[i:j+1])
            result.append(max_value)
            print("max value", max_value)
            print("i=",i)
            print("j=",j)
            i += 1
            j += 1

        return result  
        